src/pg_cron.c

```diff
@@ -521,6 +605,10 @@ PgCronLauncherMain(Datum arg)
 
 	ereport(LOG, (errmsg("pg_cron scheduler started")));
 
+	/* set the desired log_min_messages */
+	SetConfigOption("log_min_messages", cron_error_severity(CronLogMinMessages),
+										PGC_POSTMASTER, PGC_S_OVERRIDE);
```

Comment:
Is there a way to allow this GUC to be changed later?

Reply:
good point, let me update the PR to allow `cron.log_min_messages` to be changed and the bgworker launcher to take into account the change.

Reply:
done. In the new version cron.log_min_messages is defined as "PGC_SIGHUP" and the pg_cron_sighup() takes care of the reload, so that:
```
alter system set cron.log_min_messages = <NEW_VALUE>
select pg_reload_conf();
```
will be taken into account by the bgworker launcher.